tests/main/auto-refresh-private/task.yaml

```diff
@@ -52,18 +54,22 @@ execute: |
     done
 
     echo "Check they were both refreshed"
+    #shellcheck disable=SC1117
```

Comment:
Hmm, isn't this supposed to be `\\` Alternatively can we use `'test-snapd-public +2\.0'`?